### PR TITLE
Deprecate openfisca-web-api package

### PR DESCRIPTION
**Pull request by sandcha**
Connected to openfisca/openfisca-core#577

Add deprecation information to README.
Explain deprecation main reasons and add link to new API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+
+The aim of this package was to provide OpenFisca information and computations through a web interface, without installing anything locally.
+
+However, after a few years of existence, it appeared:
+* That versioning OpenFisca API and the Core module apart was difficult to maintain
+* That this API endpoints needed to be improved to match users experiences
+
+## Introduction
+
+[![Build Status](https://travis-ci.org/openfisca/openfisca-web-api.svg?branch=master)](https://travis-ci.org/openfisca/openfisca-web-api) - [More build statuses](https://www.openfisca.fr/build-status)
```
Comment (fpagnoux) [inline README.md]: We can remove this : for a deprecated package, exposing the build status is not really relevant.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,22 @@
-# OpenFisca Web-API
+# OpenFisca Web-API - DEPRECATED
 
-[![Build Status](https://travis-ci.org/openfisca/openfisca-web-api.svg?branch=master)](https://travis-ci.org/openfisca/openfisca-web-api)
+## This package is deprecated
 
-[More build statuses](https://www.openfisca.fr/build-status)
+**The official API is now available in [OpenFisca Core repository](https://github.com/openfisca/openfisca-core).**
```
Comment (fpagnoux) [inline README.md]: **The OpenFisca Web API is now packaged in [OpenFisca-Core](https://github.com/openfisca/openfisca-core).**

File: README.md [hunk 3]
```diff
@@ -1,11 +1,22 @@
-# OpenFisca Web-API
+# OpenFisca Web-API - DEPRECATED
 
-[![Build Status](https://travis-ci.org/openfisca/openfisca-web-api.svg?branch=master)](https://travis-ci.org/openfisca/openfisca-web-api)
+## This package is deprecated
 
-[More build statuses](https://www.openfisca.fr/build-status)
+**The official API is now available in [OpenFisca Core repository](https://github.com/openfisca/openfisca-core).**  
+**The current package isn't maintained anymore: it won't evolve, and it is not guaranteed to be compatible with future versions of OpenFisca.**
```
Comment (fpagnoux) [inline README.md]: Hum, this package is still used by some of our main re-users. We should try to find a balance in order to incentivize them to use the new API, without scaring them too much. I'd remove the "it won't evolve" part.

Comment (sandcha) [reply README.md]: 🙂

**Pull request merged**